core/src/main/scala/kafka/tools/MirrorMaker.scala

```diff
@@ -184,6 +191,10 @@ object MirrorMaker extends Logging with KafkaMetricsGroup {
           System.exit(1)
         }
       } else {
+        if (options.has(pollMsOpt)) {
+          error(s"$pollMsOpt can only be used with $useNewConsumerOpt.")
```

Comment:
The jira states the old consumer supports this. Can we pass this value to the old consumer too?